arangod/Cluster/MaintenanceFeature.cpp

```diff
@@ -39,6 +39,9 @@ using namespace arangodb::application_features;
 using namespace arangodb::options;
 using namespace arangodb::maintenance;
 
+const uint32_t MaintenanceFeature::minThreadLimit = 2;
```

Comment:
This has to be 3 to pass Jenkins.  There was another PR that discussed the issue and a comment on the PR saying same.

Reply:
Should also add a comment so someone like me does not come back later and screw this up.